oauth2client/_helpers.py

```diff
@@ -300,3 +295,55 @@ def _urlsafe_b64decode(b64string):
     b64string = _to_bytes(b64string)
     padded = b64string + b'=' * (4 - len(b64string) % 4)
     return base64.urlsafe_b64decode(padded)
+
+
+def update_query_params(uri, params):
```

Comment:
When a code element is both introduced to a file and used in the same file, please place the code element earlier in the file ("above") all of its use-sites.